pkgs/development/tools/misc/clojure-lsp/default.nix

```diff
@@ -1,31 +1,63 @@
-{ lib, stdenv, fetchurl, jre, makeWrapper }:
+{ lib, stdenv, fetchurl, graalvm11-ce }:
 
 stdenv.mkDerivation rec {
   pname = "clojure-lsp";
-  version = "2021.02.02-14.02.23";
+  version = "2021.02.04-02.08.58";
 
   src = fetchurl {
     url = "https://github.com/clojure-lsp/clojure-lsp/releases/download/${version}/${pname}.jar";
-    sha256 = "sha256-NwaUIaGkSB+oVd/OArp+bs01ZDAFy5JJ4ic5sMDYg8s=";
+    sha256 = "sha256-Dwi17uLSELfN+mR4dUTN2ZZYbGGzTDEWRgSaT9Irmqk=";
   };
 
   dontUnpack = true;
 
-  buildInputs = [ makeWrapper ];
+  buildInputs = [ graalvm11-ce ];
+
+  buildPhase = with lib; ''
+    args=("-jar" "${src}"
+          "-H:Name=clojure-lsp"
+          ${optionalString stdenv.isDarwin ''"-H:-CheckToolchain"''}
```

Comment:
Didn't test, just copied it from `babashka` derivation.

Reply:
Will I notice something different when building on darwin?

Reply:
Don't know, I don't have a macOS to test.